### ISSUE TITLE
Addon won't open properly

### ISSUE THREAD

**Issue by joseb16** [mathsgrinds/Kodi-Video-Plugin-Live-Irish-TV#20]
Hi, sorry to be the bearer of bad news again! The addon won't open properly on 2 kodi installs since the update to 1.05/1.06. When I click on it just shows a blank screen, like the standard blank screen from before in the few seconds before the guide opens, but now the guide/channels won't display at all. Getting log errors on both boxes.

**Comment by joseb16** [comment]
Hi, a quick update - I've reverted to 1.0.0, 2.0.9 and 2.1.3 and clear cache after but they all had some issues. Then downloaded zip made 10 minutes ago and it now shows the information for all programmes. Tried both safe mode on/off and ok does nothing. When I right click and 'play from here' it throws up a log error on every channel, and for some reason starts playing irishtv (which seems to work flawlessly).

Earlier I used iVue creator which basically scans video addons and extracts their play button for use with a tv guide. It worked fine and played every channel, though it needed to be updated every few hours due to the nature of ietvnow. This no longer works, and throws up a log error. 

Thanks again for the help, don't mean to hurry you along, but thought I'd give an update on the issue. 

As a matter of interest how has ietvnow been working on your box? Thanks.
